packages/core-flows/src/product-category/steps/create-product-category.ts

```diff
@@ -0,0 +1,35 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  CreateProductCategoryDTO,
+  IProductModuleService,
+} from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+type CreateProductCategoryStepInput = {
+  product_category: CreateProductCategoryDTO
```

Comment:
Typically we accept an array for the input, should we do the same here? Or is there a reason we can't?

Reply:
The product category service + repository will need a makeover to allow for bulk operations. This is a pretty comprehensive change, so I decided to open this PR now and then we can address the bulk support in a follow-up PR.